dev-python/pyside/pyside-9999.ebuild

```diff
@@ -1,71 +1,93 @@
 REQUIRED_USE="
 	${PYTHON_REQUIRED_USE}
-	declarative? ( X )
-	designer? ( X )
-	help? ( X )
-	multimedia? ( X )
-	opengl? ( X )
-	scripttools? ( X script )
-	sql? ( X )
-	svg? ( X )
-	test? ( X )
-	webkit? ( X )
+	declarative? ( gui network )
+	designer? ( widgets )
+	help? ( widgets )
+	multimedia? ( gui network )
+	opengl? ( widgets )
+	printsupport? ( widgets )
+	scripttools? ( gui script )
+	sql? ( widgets )
+	svg? ( widgets )
+	test? ( widgets )
+	testlib? ( widgets )
+	webchannel? ( network )
+	webengine? ( network webchannel widgets )
+	webkit? ( gui network printsupport widgets )
+	websockets? ( network )
+	widgets? ( gui )
+	xmlpatterns? ( network )
 "
 
-# Minimal supported version of Qt.
-QT_PV="4.8.5:4"
+# Minimum Qt version required, derived from the "CMakeLists.txt" line:
+#      find_package(Qt5 ${QT_PV} REQUIRED COMPONENTS Core)
+QT_PV="5.3.0:5"
 
 RDEPEND="
 	${PYTHON_DEPS}
-	>=dev-python/shiboken-${PV}[${PYTHON_USEDEP}]
+	>=dev-python/shiboken-${PV}:${SLOT}=[${PYTHON_USEDEP}]
 	>=dev-qt/qtcore-${QT_PV}
-	X? (
-		>=dev-qt/qtgui-${QT_PV}[accessibility]
-		>=dev-qt/qttest-${QT_PV}
-	)
-	declarative? ( >=dev-qt/qtdeclarative-${QT_PV} )
+	>=dev-qt/qtxml-${QT_PV}
+	concurrent? ( >=dev-qt/qtconcurrent-${QT_PV} )
+	declarative? ( >=dev-qt/qtdeclarative-${QT_PV}[widgets?] )
 	designer? ( >=dev-qt/designer-${QT_PV} )
+	gui? ( >=dev-qt/qtgui-${QT_PV} )
 	help? ( >=dev-qt/qthelp-${QT_PV} )
-	multimedia? ( >=dev-qt/qtmultimedia-${QT_PV} )
+	multimedia? ( >=dev-qt/qtmultimedia-${QT_PV}[widgets?] )
+	network? ( >=dev-qt/qtnetwork-${QT_PV} )
 	opengl? ( >=dev-qt/qtopengl-${QT_PV} )
+	printsupport? ( >=dev-qt/qtprintsupport-${QT_PV} )
 	script? ( >=dev-qt/qtscript-${QT_PV} )
 	sql? ( >=dev-qt/qtsql-${QT_PV} )
-	svg? ( >=dev-qt/qtsvg-${QT_PV}[accessibility] )
-	webkit? ( >=dev-qt/qtwebkit-${QT_PV} )
+	svg? ( >=dev-qt/qtsvg-${QT_PV} )
+	testlib? ( >=dev-qt/qttest-${QT_PV} )
+	webchannel? ( >=dev-qt/qtwebchannel-${QT_PV} )
+	webengine? ( >=dev-qt/qtwebengine-${QT_PV}[widgets?] )
+	webkit? ( >=dev-qt/qtwebkit-${QT_PV}[printsupport] )
+	websockets? ( >=dev-qt/qtwebsockets-${QT_PV} )
+	widgets? ( >=dev-qt/qtwidgets-${QT_PV} )
+	x11extras? ( >=dev-qt/qtx11extras-${QT_PV} )
 	xmlpatterns? ( >=dev-qt/qtxmlpatterns-${QT_PV} )
 "
-DEPEND="${RDEPEND}
-	>=dev-qt/qtgui-${QT_PV}
-"
-
-DOCS=( ChangeLog )
+DEPEND="${RDEPEND}"
 
 src_prepare() {
 	# Fix generated pkgconfig file to require the shiboken
 	# library suffixed with the correct python version.
 	sed -i -e '/^Requires:/ s/shiboken$/&@SHIBOKEN_PYTHON_SUFFIX@/' \
-		libpyside/pyside2.pc.in || die
+		libpyside/${PN}2.pc.in || die
+
+	# Force the optional "Qt5Concurrent", "Qt5Gui", "Qt5Network",
+	# "Qt5PrintSupport", "Qt5Sql", "Qt5Test", and "Qt5Widgets" packages
+	# erroneously marked as mandatory to be optional.
+	sed -i -e 's/^\(CHECK_PACKAGE_FOUND(Qt5\(Concurrent\|Gui\|Network\|PrintSupport\|Sql\|Test\|Widgets\)\))$/\1 opt)/' \
```

Comment:
**Absolutely.** I'm lazy, slovenly, and pressed for time at the moment. Upstream patch submission appears to be a bit of a... *process.* Cumbersome corporate bureaucracy: who would have thought?

If I fail to submit an actual patch, I'll try to at least submit a bug report suggesting these changes. :bug: